ish_parser/Speed.py

```diff
@@ -1,18 +1,20 @@
 from .Observation import Observation
 
 class Speed(Observation):
-  ''' observation of a temperature, which we can convert
-  between farenheight and celsius, yay '''
+  ''' observation of a wind speed, which we can convert
+  between m/s and mph, yay '''
   METERSPERSECOND = 1
-  MILESPERHOUR = 1
+  MILESPERHOUR = 2
+  SPEED_SCALE = 10
+  MPH_PER_MPS = 100 / 2.54 / 12 / 5280 * 60 * 60
   MISSING = [999, 9999, 999.9]
 
-  def get_miles(self):
+  def get_MilesPerHour(self):
```

Comment:
Can we change this to snake case to fit in with convention? get_miles_per_hour / get_mph